etc/adminhtml/menu.xml

```diff
@@ -9,7 +9,5 @@
     <menu>
         <add id="RedChamps_CleanMenu::extensions" title="Extensions" translate="title" module="RedChamps_CleanMenu" sortOrder="60" dependsOnModule="RedChamps_CleanMenu" resource="RedChamps_CleanMenu::extensions"/>
         <update id="Magento_Marketplace::partners" dependsOnConfig="clean_admin_menu/marketplace/enabled" dependsOnModule="Magento_Marketplace" title="Find Partners &amp; Extensions" translate="title" resource="Magento_Marketplace::partners"/>
-        <!--This is to fix conflict with Amasty_Base Extension-->
-        <remove id="Amasty_Base::marketplace" />
```

Comment:
I have tried extension installation on clean Magento 2.4.1 instance and it doesn't trigger any error. Also, we haven't received any complaint about this problem.

Reply:
this breaks on M2.2

Reply:
M2.2 is eol so I agree let's ignore this, I've reverted this change.